tensorboard/program.py

```diff
@@ -341,79 +342,128 @@ def __init__(self, msg):
     self.msg = msg
 
 
+class TensorBoardFailedToBindError(TensorBoardServerException):
```

Comment:
Given that this error actually shouldn't be raised for all failures to bind, maybe it's better to just make this TensorBoardPortInUseError?  That's the only case that we actually want to retry.  Or at a minimum something like TensorBoardRetryableBindError.

Reply:
Both good suggestions, thanks. I’ll go with “port in use” but have no
strong preference.